docs/source/core.rst

```diff
@@ -353,6 +411,30 @@ Returns allowance (tokens left to use by spender) for given tokenOwner - spender
     {allowance && <p>Remaining allowance: {formatUnits(allowance, 18)} tokens</p>}
   )
 
+.. _useTransactions:
+
+useTransactions
+===============
+
+``useTransactions`` hook returns a list ``transactions`` of transactions sent from useDApp.
```

Comment:
list of what transactions? Perhaps all transactions that were triggered with useContractFunction or useSendTransaction?